### PR TITLE
Fix/html formhelper keys

### PR DESCRIPTION
**Pull request by peterjanssens**
some value initialisation problems, mixed with wrong keys (value vs values)

### CODE REVIEW

File: app/helpers/htmlFormHelpers.php [hunk 1]
```diff
@@ -76,7 +78,7 @@ function caHTMLSelect($ps_name, $pa_content, $pa_attributes=null, $pa_options=nu
 		
 		$vs_element = "<select name='{$ps_name}' {$vs_attr_string}>\n";
 		
-		$vs_selected_val = isset($pa_options['value']) ? $pa_options['value'] : null;
```
Comment (collectiveaccess) [inline app/helpers/htmlFormHelpers.php:79]: This is there to allow a single value to be passed via the "value" option. Multiple values can be passed with the "values" option.

**Comment by collectiveaccess** [comment]
Thanks for this one. The "value" vs. "values" key differences are intentional and required to support both "value" and "values" options, the former being for a single selected value. 

I'm going to merge in your initialization and add comments to explain what's going on in this bit of code.

**Comment by peterjanssens** [comment]
thx for the update, some things like that are not always clear without (historical) context :)

**Comment by collectiveaccess** [comment]
In this case it's very old history (like from ~2005!)

**Pull request closed without merging**